src/app/views/homepage/homepage.component.html

```diff
@@ -44,28 +48,35 @@ <h1 class="card-header">
-        <li><a class="nav-link" routerLink="/events/workshop-and-meetings" href="#" onclick="window.scrollTo(0, 0)">Seminars</a></li>
-      </ul>
-    </span>
+
+<section class="p-3" style="background-image: url('https://images.rawpixel.com/image_800/czNmcy1wcml2YXRlL3Jhd3BpeGVsX2ltYWdlcy93ZWJzaXRlX2NvbnRlbnQvbHIvdjk5Ni0wMDlfMS1rcm9pcjRkay5qcGc.webp');">
+  <div class="row" style="z-index: 2;">
+    <div class="col-md-6 px-md-5 me-2 pt-4 mb-5"><h3 class="text-center mb-5" style="text-decoration: underline; font-weight: bold;">Programs offered by ICWaR</h3>
+      <p class="px-md-5" style="text-align: justify;">
+        <b>•	PhD in Water Research</b><br>
+            We offer a PhD program in Water Research.<br><br>
+        <b>•	Postdoctoral Positions</b><br>
+            Prospective Postdocs, INSPIRE Fellows, CV Raman Postdocs, and NPDFs are encouraged to contact our faculty members directly to discuss potential opportunities. <br><br>
+        <b>•	Project Positions</b><br>
+        We offer a variety of project-based positions in Water Research. Specific positions available will vary based on the current project needs and requirements.
+      </p><br>
+      <div class="text-center px-md-5"><h3 class="mb-4"><b>Are you interested to join us?</b></h3>
+        Find details on eligibility, application deadlines, and submission guidelines to take the next step in your research career
+        <a style="color: rgb(34, 163, 146); font-size: large;" routerLink="/join-icwar" onclick="window.scrollTo({ top: 0, behavior: 'smooth' });"><b>&#64;Join ICWaR</b></a></div>
+    </div>
+
     <div class="vr d-md-block d-none p-0"></div>
-    <div class="col-3 d-md-block d-none offset-md-1 justify-content-center align-items-center" style="font-size: calc(1rem + 0.5vw);" >
-      "Opportunities don't happen, you create them."
+    <div class="col-md-5 ps-md-5 ms-md-5 mb-5 mt-5 mt-md-0 text-center"><img src="../../../assets/images/cCauvery_ICWaR.webp" alt="" style="max-height: 20vh; max-width: 100%;" class="align-self-center mb-3">
+      <p style="text-align: justify;">The river Cauvery is uniquely essential to India for geographical, historical, socio-cultural, and economic reasons. In this reference, the Centre for Cauvery River Basin Management and Studies (<a style="color: rgb(0, 115, 192);" target="_blank" href="https://krishnapandeykp.github.io/cCauvery"><b>cCauvery</b></a>), a brain trust dedicated to river science and river basin management, was established in 2024 by IISc Bengaluru and NIT Tiruchirappalli, under the supervision of cGanga at IIT Kanpur, the centre serves as a knowledge wing of the National River Conservation Directorate (NRCD). cCauvery is committed to restoring and conserving the Cauvery River and its resources through collating information and knowledge, research and development, planning, monitoring, education, advocacy, and stakeholder engagement.</p>
```

Comment:
https://krishnapandeykp.github.io/cCauvery
This site is good work. Just confirm once if deployment and redirect to your personal github account is allowed in the department